### DISCUSSION TITLE
Is there anyway to stop having Phoenix give me little errors that don't really matter?

### DISCUSSION THREAD

**Discussion by treeguy1234** [phcode-dev/phoenix#1742] [Q&A] [answered]
Phoenix keeps giving me errors that are not necessary. Is there any way to stop that?

<img width="782" alt="image" src="https://github.com/phcode-dev/phoenix/assets/125884868/ec298127-75c4-4bd2-9494-7cef5ae3abdc">
<img width="781" alt="image" src="https://github.com/phcode-dev/phoenix/assets/125884868/7c378689-13ae-4b83-acfb-bb9f8c71969d">

**Comment by abose** [comment, marked as answer]
To permanently disable HTML and js error hints, follow these steps:

1. Go to `File` > `Preferences`.
2. Open the `phcode.json` file in the right pane.
3. Add the following lines to the file:

```json
"HTMLLint.disabled": true,
"jshint.disabled": true
```

This will disable error detection for both HTML and JavaScript files.
